### PR TITLE
AVIRIS support in .atcorr

### PR DESCRIPTION
**Pull request by YannChemin**
AVIRIS is a plane flying about 20Km high, the 6S configuration file should have three information after the sensor height, in three different lines: 
- puw (water vapor content between the aircraft and the surface)
- po3 (ozone content between the aircraft and the surface)
- taerp (the aerosol optical thickness at 550nm between the aircraft and the surface)

If you do not have this information, please use -1.0

Example
+++++++

31
01 16 23.725 -138.90 28.90
1
2
18
-0.6
-20.56
-1.0
-1.0
-1.0
367

### CODE REVIEW

**Comment by nilason** [comment]
@YannChemin The ClangFormat check is failing, the code need to be formatted with ClangFormat 15. See https://github.com/OSGeo/grass/blob/main/doc/development/submitting/submitting_c.md#source-code-formatting, I recommend using pre-commit for doing this https://github.com/OSGeo/grass/blob/main/doc/development/submitting/submitting.md#use-pre-commit.

**Comment by nilason** [comment]
> > @YannChemin The ClangFormat check is failing, the code need to be formatted with ClangFormat 15. See https://github.com/OSGeo/grass/blob/main/doc/development/submitting/submitting_c.md#source-code-formatting, I recommend using pre-commit for doing this https://github.com/OSGeo/grass/blob/main/doc/development/submitting/submitting.md#use-pre-commit.
> 
> Hi how to run it again on already committed code?

```pre-commit run clang-format --all-files```

File: imagery/i.atcorr/altitude.cpp [hunk 1]
```diff
@@ -382,6 +382,8 @@ void Altitude::parse()
     cin >> original_xpp;
     cin.ignore(numeric_limits<int>::max(), '\n'); /* ignore comments */
     original_xpp = -original_xpp;
+    printf("Original Alt: XPS = %.2f\t XPP = %.2f\n", original_xps,
```
Comment (nilason) [inline imagery/i.atcorr/altitude.cpp:385]: Please consider using `G_message()` (see https://github.com/OSGeo/grass/blob/main/doc/development/submitting/submitting_c.md#messages).

File: imagery/i.atcorr/iwave.cpp [hunk 2]
```diff
@@ -4829,7 +6463,7 @@ void IWave::parse()
 
     cin >> iwave;
     cin.ignore(numeric_limits<int>::max(), '\n');
-
+    printf("iwave = %d\n", iwave);
```
Comment (nilason) [inline imagery/i.atcorr/iwave.cpp:6466]: Ditto, please consider using `G_message()`.

**Review by nilason — COMMENTED** [review]
Glad you sorted out the clang format issue. I have noted two minor suggestions.

File: imagery/i.atcorr/altitude.cpp (hunk 1 above)
Comment (YannChemin) [reply imagery/i.atcorr/altitude.cpp:385]: There is a G_Debug() maybe?

Comment (nilason) [reply imagery/i.atcorr/altitude.cpp:385]: Yes, using `G_debug()` seems perhaps even more appropriate in this case
(https://grass.osgeo.org/programming8/debug_8c.html#af8982bd66c95e3d8d2773ed956c20733).

Comment (YannChemin) [reply imagery/i.atcorr/altitude.cpp]: Hi what do I do now?

Comment (neteler) [reply imagery/i.atcorr/altitude.cpp]: Use `G_debug()` :-)

**Pull request merged**